### PR TITLE
[PTMEX042023]-Walter-Bernal

### PR DESCRIPTION
**Pull request by WaltBernalM**
done

### CODE REVIEW

File: src/clue.js [hunk 1]
```diff
@@ -1,26 +1,216 @@
...
+  mrsPeacock,
+  mrMustard
+]
 
-const suspectsArray = [];
 
 // Rooms Array
 
-const roomsArray = [];
+let diningRoom = {
```
Comment (ta-web-mexpt2) [inline src/clue.js]:
```suggestion
 {name: "Dining Room"},
 {name: "Conservatory"},
    {name: "Kitchen"},
```
inside of rooms array and you are not to set the variables for all the rooms the same with the others

File: src/clue.js [hunk 2]
```diff
@@ -1,26 +1,216 @@
...
+  observatory,
+  theater,
+  guestHouse,
+  patio
+]
 
 // Weapons Array
 
-const weaponsArray = [];
+let rope = {
```
Comment (ta-web-mexpt2) [inline src/clue.js]: same here

```suggestion
 {name: "rope",
 weight: 10,}
```

File: src/clue.js [hunk 3]
```diff
@@ -1,26 +1,216 @@
...
+    room: selectRandom(roomsArray),
+    weapon: selectRandom(weaponsArray)
+  }
+}
 
 
 // ITERATION 3
 
-function revealMystery() {}
+function revealMystery(object) {
```
Comment (ta-web-mexpt2) [inline src/clue.js]: better leave the @param with sintactic sugar 

```suggestion
function revealMystery(mistery) {
```

**Review by ta-web-mexpt2 — CHANGES_REQUESTED** [review]

**Review by WaltBernalM — COMMENTED** [review]
I've refactored all the code to follow your feedback

**Pull request closed without merging**